IntelFsp2Pkg/FspSecCore/SecFsp.c

```diff
@@ -147,7 +147,6 @@ FspGlobalDataInit (
   // It may have multiple FVs, so look into the last one for FSP header
   //
   PeiFspData->FspInfoHeader = (FSP_INFO_HEADER *)(UINTN)AsmGetFspInfoHeader ();
-  SecGetPlatformData (PeiFspData);

```

Comment:
May I know why it's removed? If we don't need this, SecGetPlatformData function should be removed from this file too.